src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -328,6 +328,26 @@
             -n MyHttpSettings --probe MyNewProbe
 """
 
+helps['network application-gateway identity'] = """
+type: group
+short-summary: Manage the identity information
+"""
+
+helps['network application-gateway identity assign'] = """
+type: command
+short-summary: Assign a managed service identity to the application-gateway
+"""
+
+helps['network application-gateway identity remove'] = """
+type: command
+short-summary: Remove a managed service identity to the application-gateway
+"""
+
+helps['network application-gateway identity show'] = """
+type: command
+short-summary: Show a managed service identity of the application-gateway
+"""
```

Comment:
You should provider examples for identity commands usage.

Reply:
Good point. I will add examples